### PR TITLE
[FIX] account: always show sales tax on product form

### PR DESCRIPTION
**Pull request by shrm-odoo**
Before this fix:
Sales tax was only shown on products marked as 'saleable', causing issues for products used for discounts, shipping, etc., which need default tax regardless of sales status.

After this fix:
Sales Tax is always displayed on the product form.

Task:4008793

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by kcv-odoo — APPROVED** [review]
You should probably change [this line](https://github.com/odoo/odoo/blob/master/addons/point_of_sale/views/product_view.xml#L112-L117) since main label and div does not contain any invisible attribute on then here it should add direct invisible condition instead OR [like this](https://github.com/odoo/odoo/blob/master/addons/point_of_sale/views/product_view.xml#L97-L99)

May be keeping as it is might be better idea so if invisible coming from other module and we install POS after that we don't overwrite that value :thought_balloon:

**Comment by shrm-odoo** [comment]
> You should probably change [this line](https://github.com/odoo/odoo/blob/master/addons/point_of_sale/views/product_view.xml#L112-L117) since main label and div does not contain any invisible attribute on then here it should add direct invisible condition instead OR [like this](https://github.com/odoo/odoo/blob/master/addons/point_of_sale/views/product_view.xml#L97-L99)
> 
> May be keeping as it is might be better idea so if invisible coming from other module and we install POS after that we don't overwrite that value 💭

Invisible attribute for taxes_id and taxes_div is not coming from any other module, so i think we can directly add invisible condition in POS.

**Comment by kcv-odoo** [comment]
I know currently there isn't any invisible attribute on `sales_div`
I meant if someone add it in the future or come from any custom module :see_no_evil:
but I am not so sure about that so we will wait for the final review opinion ;)

**Review by Feyensv — APPROVED** [review]
@robodoo r+

I think we can keep hiding the taxes for combo products for now (it's working fine and we can still bring it back later in another PR if needed).

**Pull request closed without merging**
